From 019e50be156d120fa92fc82646ec42ec963da984 Mon Sep 17 00:00:00 2001
From: Wyon Bi <bivvy.bi@rock-chips.com>
Date: Fri, 25 Sep 2020 17:05:05 +0800
Subject: [PATCH] dt-bindings: display/rockchip: Add compatible string for
 rk3568

Add a compatible string for the eDP controller found in the RK3568 SoC.

Change-Id: I4ece0815efd7d603ee0c5f5adac8d3bf5b91130e
Signed-off-by: Wyon Bi <bivvy.bi@rock-chips.com>
---
 .../devicetree/bindings/display/bridge/analogix_dp.txt       | 1 +
 .../bindings/display/rockchip/analogix_dp-rockchip.txt       | 5 +++--
 2 files changed, 4 insertions(+), 2 deletions(-)

diff --git a/Documentation/devicetree/bindings/display/bridge/analogix_dp.txt b/Documentation/devicetree/bindings/display/bridge/analogix_dp.txt
index 2124c58bf13b..732ef0923cfd 100644
--- a/Documentation/devicetree/bindings/display/bridge/analogix_dp.txt
+++ b/Documentation/devicetree/bindings/display/bridge/analogix_dp.txt
@@ -7,6 +7,7 @@ Required properties for dp-controller:
 		 * "rockchip,rk3288-dp"
 		 * "rockchip,rk3368-edp"
 		 * "rockchip,rk3399-edp"
+		 * "rockchip,rk3568-edp"
 	-reg:
 		physical base address of the controller and length
 		of memory mapped region.
diff --git a/Documentation/devicetree/bindings/display/rockchip/analogix_dp-rockchip.txt b/Documentation/devicetree/bindings/display/rockchip/analogix_dp-rockchip.txt
index dc40aa715c77..a9198508bbdb 100644
--- a/Documentation/devicetree/bindings/display/rockchip/analogix_dp-rockchip.txt
+++ b/Documentation/devicetree/bindings/display/rockchip/analogix_dp-rockchip.txt
@@ -5,6 +5,7 @@ Required properties:
 - compatible: "rockchip,rk3288-dp",
 	      "rockchip,rk3368-edp",
 	      "rockchip,rk3399-edp";
+	      "rockchip,rk3568-edp";
 
 - reg: physical base address of the controller and length
 
@@ -22,8 +23,6 @@ Required properties:
 
 - reset-names: Must include the name "dp"
 
-- rockchip,grf: this soc should set GRF regs, so need get grf here.
-
 - ports: there are 2 port nodes with endpoint definitions as defined in
   Documentation/devicetree/bindings/media/video-interfaces.txt.
     Port 0: contained 2 endpoints, connecting to the output of vop.
@@ -34,7 +33,9 @@ Optional property for different chips:
 
 - clock-names: from common clock binding:
 	       Required elements: "grf"
+- rockchip,grf: this soc should set GRF regs, so need get grf here.
 Optional properties
+- reset-names: "apb" must control the reset line to the APB interface.
 - vcc-supply: Regulator for eDP_AVDD_1V0.
 - vccio-supply: Regulator for eDP_AVDD_1V8.
 
-- 
2.35.3

